src/dashboard/Data/Browser/DataBrowser.react.js

```diff
@@ -426,26 +539,72 @@ export default class DataBrowser extends React.Component {
     const { preventSchemaEdits, applicationId } = app;
     return (
       <div>
-        <BrowserTable
-          appId={applicationId}
-          order={this.state.order}
-          current={this.state.current}
-          editing={this.state.editing}
-          simplifiedSchema={this.state.simplifiedSchema}
-          className={className}
-          editCloneRows={editCloneRows}
-          handleHeaderDragDrop={this.handleHeaderDragDrop}
-          handleResize={this.handleResize}
-          setEditing={this.setEditing}
-          setCurrent={this.setCurrent}
-          setCopyableValue={this.setCopyableValue}
-          setContextMenu={this.setContextMenu}
-          onFilterChange={this.props.onFilterChange}
-          onFilterSave={this.props.onFilterSave}
-          selectedCells={this.state.selectedCells}
-          handleCellClick={this.handleCellClick}
-          {...other}
-        />
+        <div>
+          <BrowserTable
+            appId={applicationId}
+            order={this.state.order}
+            current={this.state.current}
+            editing={this.state.editing}
+            simplifiedSchema={this.state.simplifiedSchema}
+            className={className}
+            editCloneRows={editCloneRows}
+            handleHeaderDragDrop={this.handleHeaderDragDrop}
+            handleResize={this.handleResize}
+            setEditing={this.setEditing}
+            setCurrent={this.setCurrent}
+            setCopyableValue={this.setCopyableValue}
+            selectedObjectId={this.state.selectedObjectId}
+            setSelectedObjectId={this.setSelectedObjectId}
+            callCloudFunction={this.props.callCloudFunction}
+            setContextMenu={this.setContextMenu}
+            onFilterChange={this.props.onFilterChange}
+            onFilterSave={this.props.onFilterSave}
+            selectedCells={this.state.selectedCells}
```

Comment:
At some point we lost recently introduced [descriptive statistics](https://github.com/parse-community/parse-dashboard/pull/2529) feature (which was later fixed https://github.com/parse-community/parse-dashboard/pull/2581).

![Descriptive statistics](https://github.com/user-attachments/assets/062c97f0-6ad0-4fd3-80d9-cac7e82c9076)

It's still working fine on `alpha` branch.

P.S. @mtrezza it's another time when this feature gets broken. I think it shows some weakness and I would blame they way selection data is structured, updated and passed between components. Maybe it would be good to do some refactoring and abstract all selection logic into single place.

Reply:
@404-html refactor optimization is always welcome, but you mean in separate PR, right?

Reply:
>At some point we lost recently introduced https://github.com/parse-community/parse-dashboard/pull/2529 feature (which was later fixed https://github.com/parse-community/parse-dashboard/pull/2581). Just try to select cells with shift pressed.

okay sure i will try find and resolve why it is failing in this PR

Reply:
> @404-html refactor optimization is always welcome, but you mean in separate PR, right?

Correct. It might be quite a big change, which will produce a lot of conflicts in existing PRs.